### ISSUE TITLE
Try using weight decay

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/CancerDetection#6]
We could try using weight decay to reduce overfitting.
